Options.php

```diff
@@ -59,8 +59,9 @@ public function get_key() {
 	 * @return mixed Whatever is in those fields.
 	 */
 	public function get( $field = null, $default = null ) {
+		$option_array = get_option( $this->key, array() );
+		$option_array = gettype($option_array) != "array" ? [] : $option_array;
```

Comment:
Short array syntax `[]` available since PHP 5.4, what is doesn't fit the WordPress' minimal requirements.